HomeWorks/Homework12Nbrb/TMS.Homework.Nbrb/UILibrary/Command.cs

```diff
@@ -10,18 +10,22 @@ class Command
         public string Param;
 
         public Command(string commandLine)
-            
-        { 
+
+        {
             ParseCommandLine(commandLine, out Name, out Param);
         }
 
         private void ParseCommandLine(string commandLine, out string name, out string param)
         {
-           string[] NameParamArray = commandLine.Split('-');
+            string[] NameParamArray = commandLine.Split('-');
+
+            if (NameParamArray.Length != 0)
+            {
+                name = NameParamArray[0].ToLower();
```

Comment:
UI will not process the following command to match "list" this case: "list -n", you should trim the name